apps/vaults/utils/actions.ts

```diff
@@ -25,12 +25,12 @@ import type {TWriteTransaction} from '@common/utils/toWagmiProvider';
 ** @param amount - The amount of the underlying asset to deposit.
 ******************************************************************************/
 type TDepositAndStake = TWriteTransaction & {
-	vaultAddress: TAddressWagmi;
+	vaultAddress: TAddress | undefined;
```

Comment:
nit: Can be like this for now, but usually we want to never explicitly assign anything to `undefined` so if optional I'd just do 
```suggestion
	vaultAddress?: TAddress;
```
and, if we want it to be an empty value then
```suggestion
	vaultAddress: TAddress | null;
```